### PR TITLE
Update resource names formatter helpers

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Update resource names formatter helpers

#### Description

In this PR we update the helper functions we use to format entry point URI paths to resource access names so it uses simplified output names.

By now we're only updating the logic for the upcoming Custom Views feature.
The default resource access group names were generated with the ID of the Custom View, which we are dropping.
```
// BEFORE
view12345
manage12345

// AFTER
view
manage
```

In the future, we plan to also apply this change to Custom Applications (in a backwards compatible way).

### CODE REVIEW

**Review by kark — APPROVED** [review]
Thank you, looks good to me 🙌

**Review by chloe0592 — APPROVED** [review]
Makes sense to me. Thank you!

**Review by ddouglasz — APPROVED** [review]
Thank you 🙌🏽

**Review by Rhotimee — APPROVED** [review]
Thanks, looks good

**Review by emmenko — COMMENTED** [review]
Many thanks! 🙌

File: packages/application-config/src/formatters.ts [hunk 1]
```diff
@@ -88,8 +89,11 @@ function entryPointUriPathToResourceAccesses<
   entryPointUriPath: string,
   permissionGroupNames?: PermissionGroupName[]
 ): TImplicitCustomApplicationResourceAccesses<PermissionGroupName> {
-  const resourceAccessKey =
-    formatEntryPointUriPathToResourceAccessKey(entryPointUriPath);
+  const resourceAccessKey = formatEntryPointUriPathToResourceAccessKey(
+    CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH === entryPointUriPath
+      ? ''
+      : entryPointUriPath
+  );
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply packages/application-config/src/formatters.ts]: The outcome it's the same, but I like not calling the format function if it's not needed.

Updated here: f445202e2142d330e2b91ad32bede9190d36cbe8

File: packages/application-config/src/formatters.ts [hunk 2]
```diff
@@ -128,9 +132,12 @@ function entryPointUriPathToPermissionKeys<PermissionGroupName extends string>(
 ): TImplicitCustomApplicationPermissionKeys<PermissionGroupName> {
   const resourceAccesses =
     entryPointUriPathToResourceAccesses<PermissionGroupName>(
-      entryPointUriPath,
+      CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH === entryPointUriPath
+        ? ''
+        : entryPointUriPath,
```
[the remark this answers is not in the document]
Comment (CarlosCortizasCT) [reply packages/application-config/src/formatters.ts]: You're totally right.

Updated here: f445202e2142d330e2b91ad32bede9190d36cbe8

**Review by emmenko — APPROVED** [review]
Wonderful 🤗

**Pull request merged**